CHANGES.md

```diff
@@ -4510,3 +4510,5 @@ _This releases fixes 2D and other issues with Chrome 29.0.1547.57 ([#1002](https
 ### a0 - 2011-06-17
 
 - Initial Release.
+- 
+-fixed a typo in line 88 of PolylineCommon.glsl 
```

Comment:
This change should be listed at the top of the file, under "Fixes" in the 1.108 release. 
Also, I don't think this is really a typo. The old code was correct, but your code will be faster. Perhaps you could say "optimized angle calculations in Polyline shader"